### DISCUSSION TITLE
Domain not improperly configures

### DISCUSSION THREAD

**Discussion by ehosseiniasl** [community/community#35837] [Other Feature Feedback, Questions, & Ideas]
I have verified my domain with github.
When adding the domain to a specific github project page, it shows below message

github project:  https://github.com/ehosseiniasl/new_website

<img width="801" alt="Screen Shot 2022-10-11 at 4 43 05 PM" src="https://user-images.githubusercontent.com/6238049/195218064-1c834cfe-d98d-4363-b8a2-1f53a34bd12a.png">

**Comment by MitkoVtori** [comment]
It's probably because you use a fork, I don't know. It also could be because from that I see in the links the website is named jonabron.info or something. You can try finding help here: https://support.github.com/
